core/src/main/java/org/elasticsearch/search/aggregations/bucket/nested/ReverseNestedAggregationBuilder.java

```diff
@@ -82,7 +88,30 @@ public String path() {
     @Override
     protected AggregatorFactory<?> doBuild(AggregationContext context, AggregatorFactory<?> parent, Builder subFactoriesBuilder)
             throws IOException {
-        return new ReverseNestedAggregatorFactory(name, type, path, context, parent, subFactoriesBuilder, metaData);
+        NestedScope nestedScope = context.searchContext().getQueryShardContext().nestedScope();
+        if (nestedScope.getObjectMapper() == null) {
```

Comment:
good catch! This would also fail if the parent `nested` agg was unmapped. I fixed this and adjusted a test too: 222aeb0d97d1946592d7281f93c65678ac5bcada